### PR TITLE
Cldc 1307 supported housing scheme locations

### PR DESCRIPTION
**Pull request by moarpheus**
Screenshot 2022-06-15 at 14 10 06

### CODE REVIEW

File: app/models/location.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Location < ApplicationRecord
+  belongs_to :scheme
+
+  WHEELCHAIR_ADAPTATION = {
```
Comment (baarkerlounger) [inline app/models/location.rb]: Should this be an enum? (The benefits are it converts between DB and display/console value for you, and it enforces that saved value must be one of those)

File: app/views/schemes/locations.html.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+<%= govuk_back_link(href: request.referer.to_s) %>
+
+<%= render partial: "organisations/headings", locals: { main: @scheme.service_name, sub: nil } %>
+
+<%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, @scheme.locations.count.eql?(1) ? "1 location" : "#{@scheme.locations.count} locations")) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <% @locations.each do |location| %>
+      <section class="x-govuk-summary-card govuk-!-margin-bottom-6">
```
Comment (baarkerlounger) [inline app/views/schemes/locations.html.erb:13]: Should this be using a govuk component? @paulrobertlloyd ?

**Review by baarkerlounger — APPROVED** [review]

File: app/models/location.rb (hunk 1 above)
Comment (moarpheus) [reply app/models/location.rb]: Will definitely have a look

File: app/views/schemes/locations.html.erb (hunk 2 above)
Comment (paulrobertlloyd) [reply app/views/schemes/locations.html.erb:13]: It’s not a GOV.UK component (yet!) but in the interim, we should add our own component, as we’re using this pattern in 2 places now (and there’s a ticket to use it in a third).

Comment (moarpheus) [reply app/views/schemes/locations.html.erb:13]: Not sure where is the other place, but will be happy to extract into separate component in the *next iteration? @baarkerlounger @paulrobertlloyd

Comment (paulrobertlloyd) [reply app/views/schemes/locations.html.erb:13]: Currently used on the review log page (i.e. https://dluhc-core-staging.london.cloudapps.digital/logs/13828/review, https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/06adfbaf6d692a282facee6726a76d3d91d245d3/app/views/form/review.html.erb), and [CLDC-1310](https://digital.dclg.gov.uk/jira/browse/CLDC-1310) is for using summary cards for listing tenants.

Component should take all the properties that can be passed to the summary list component in addition to a header title and header actions.

**Pull request merged**